samcli/local/docker/lambda_container.py

```diff
@@ -169,17 +174,20 @@ def _get_entry_point(runtime, debug_options=None):  # pylint: disable=too-many-b
         Dockerfile. We override this default specifically when enabling debugging. The overridden entry point includes
         a few extra flags to start the runtime in debug mode.
 
-        :param string runtime: Lambda function runtime name
-        :param int debug_port: Optional, port for debugger
-        :param string debug_args: Optional additional arguments passed to the entry point.
+        :param string runtime: Lambda function runtime name.
+        :param DebugContext debug_options: Optional. Debug context for the function (includes port, args, and path).
         :return list: List containing the new entry points. Each element in the list is one portion of the command.
             ie. if command is ``node index.js arg1 arg2``, then this list will be ["node", "index.js", "arg1", "arg2"]
         """
 
         if not debug_options:
             return None
 
-        debug_port = debug_options.debug_port
+        debug_ports = debug_options.debug_ports
+        if not debug_ports:
```

Comment:
This check is fine but I think it is not needed. `__bool__` on DebugContext is `bool(self.debug_ports)`. This should never execute given the first if statement (`if not debug_options`).